node/network/bitfield-distribution/src/tests.rs

```diff
@@ -471,7 +470,7 @@ fn delay_reputation_change() {
 	let pool = sp_core::testing::TaskExecutor::new();
 	let (ctx, mut handle) = make_subsystem_context::<BitfieldDistributionMessage, _>(pool);
 	let mut rng = dummy_rng();
-	let reputation_interval = Duration::from_millis(1);
+	let reputation_interval = Duration::from_millis(100);
```

Comment:
```suggestion
	let reputation_interval = Duration::from_secs(1);
```

Can we bump it even more?

Reply:
I don't see a point. It will increase the test time by a whole second. 100ms are already too much. Actual execution time, I think, is about 2ms.

Reply:
I don't suppose we could make it so that it doesn't require any timeout at all? By, for example, waiting for something that we know must happen instead of just waiting for a predefined amount of time.

Reply:
No, it's not just a timeout for tests. Here we determine a value to use in the subsystem. 
https://github.com/paritytech/polkadot/blob/1346281e1a12958bb08d5fcf55c7563750719388/node/network/bitfield-distribution/src/lib.rs#L206-L213